dts/arm/nxp/nxp_lpc55S3x_common.dtsi

```diff
@@ -247,6 +247,7 @@
 		offset-value-a = <10>;
 		offset-value-b = <10>;
 		#io-channel-cells = <1>;
+		required-regulators = <&vref0>;
```

Comment:
you should be using `*-supply` properties to list regulator dependencies.

Reply:
I will look into this